site/content/en/docs/Installation/Terraform/gke.md

```diff
@@ -92,6 +92,7 @@ Configurable parameters:
 - network - the name of the VPC network you want your cluster and firewall rules to be connected to (default is "default")
 - log_level - possible values: Fatal, Error, Warn, Info, Debug (default is "info")
 - feature_gates - a list of alpha and beta version features to enable. For example, "PlayerTracking=true&ContainerPortAllocation=true"
+- kubernetes_version - a Kubernetes version of the GKE cluster
```

Comment:
This is where we need a big warning if we add this as a variable.

Reply:
Will remove this, we don't need this for examples configuration.